patches/chrome-browser-ui-views-translate-translate_bubble_view.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/browser/ui/views/translate/translate_bubble_view.cc b/chrome/browser/ui/views/translate/translate_bubble_view.cc
+index 4052cba5e7dc..2eeea20250cf 100644
+--- a/chrome/browser/ui/views/translate/translate_bubble_view.cc
++++ b/chrome/browser/ui/views/translate/translate_bubble_view.cc
+@@ -103,6 +103,7 @@ views::Widget* TranslateBubbleView::ShowBubble(
+     translate::TranslateStep step,
+     translate::TranslateErrors::Type error_type,
+     DisplayReason reason) {
++  return nullptr; // feature disabled in Brave
```

Comment:
I realize this is a lot more work but could we try to do this without a patch?

It would look something like...

Do a chromium_src override for: chrome/browser/ui/cocoa/browser_window_controller.mm
Moving away ShowTranslateBubbleViews to ShowTranslateBubbleViews_ChromiumImpl via #define and #undef at the end.
Replace with your own function that does nothing.


Do a chromium_src override for: chrome/browser/ui/views/toolbar/toolbar_view.cc
Moving away TranslateBubbleView and replacing with your own subclass that has a static function of its own called ShowBubble.
Your class will derive from TranslateBubbleView though.